data_model/src/permission.rs

```diff
@@ -149,13 +149,6 @@ impl JsonString {
         Self(payload.to_string())
     }
 }
-impl PartialEq for JsonString {
```

Comment:
In the removed PartialEq impl these jsons would be equal: `{"key":"value"}` == `{ "key" : "value" }`.
Is it ok that they're only compared by their `String` representations?